plugins/net_plugin/net_plugin.cpp

```diff
@@ -4417,6 +4442,21 @@ namespace eosio {
          } else {
             ++num_peers;
          }
+         if((*it)->remote_endpoint_ipv4) {
+            per_connection.addresses.push_back((*it)->remote_endpoint_ip_integer);
+            per_connection.ports.push_back((*it)->remote_endpoint_port);
+            per_connection.accepting_blocks.push_back((*it)->is_blocks_connection());
+            per_connection.last_received_blocks.push_back((*it)->get_last_received_blk_num());
+            per_connection.first_available_blocks.push_back((*it)->get_peer_start_block_num());
+            per_connection.last_available_blocks.push_back((*it)->get_peer_head_block_num());
+            per_connection.bytes_received.push_back((*it)->get_bytes_received());
+            per_connection.bytes_sent.push_back((*it)->get_bytes_sent());
+            per_connection.connection_start_times.push_back((*it)->construction_time);
+            per_connection.unique_first_block_counts.push_back((*it)->get_unique_blocks_rcvd_count());
+            per_connection.latencies.push_back((*it)->get_peer_ping_time_ns());
+         }
+         else
+            fc_wlog(logger, "socket remote endpoint is not IPv4");
```

Comment:
Doesn't net_plugin support ipv6 now? Why would this be a wlog?

Reply:
Yes, net_plugin supports ipv6. The Prometheus support also needs to support ipv6.

Reply:
There are still comments in net_plugin saying ipv6 support is needed.

Reply:
Removed.